lib/rule-context.js

```diff
@@ -145,7 +157,9 @@ RuleContext.prototype = {
             location,
             message,
             opts,
-            this.meta
+            fix,
```

Comment:
I think it was on purpose that `fix` was not being passed in old style call. @ilyavolodin do you remember?